acceptance/tests/failover/intentional_shutdown_failover.rb

```diff
@@ -0,0 +1,50 @@
+require 'pxp-agent/config_helper.rb'
+require 'pxp-agent/test_helper.rb'
+
+test_name 'C97934 - agent should use next broker if primary is intentionally shutdown'
+
+teardown do
+  broker_instance = 0 # 0 indexed
+  kill_pcp_broker(master)
+  run_pcp_broker(master, broker_instance)
+end
+
+def pxp_config_hash_using_puppet_certs_multiple_uris(broker, agent)
```

Comment:
agreed, that's why i abstracted it... somewhat.  but yagni rules apply, so i left it here